src/otx/algo/segmentation/litehrnet.py

```diff
@@ -558,6 +558,9 @@ def _optimization_config(self) -> dict[str, Any]:
     @property
     def _exporter(self) -> OTXModelExporter:
         """Creates OTXModelExporter object that can export the model."""
+        if self.image_size is None:
+            raise ValueError(self.image_size)
```

Comment:
Could you update an error message including other tasks?

Reply:
Updated

Reply:
Actually, it seems like this hasn't been updated.

Reply:
Updated this time, last time I've done that for all other models